worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -717,7 +717,7 @@ private[celeborn] class Worker(
       applicationId: String = null,
       resourceConsumption: ResourceConsumption = null): Unit = {
     var resourceConsumptionLabel = userIdentifier.toMap
-    if (applicationId != null)
+    if (applicationId != null && conf.metricsPushApplicationId)
```

Comment:
```suggestion
    if (applicationId != null)
```

Reply:
I am fine to set the default top resource consumption count to 0. thanks

Reply:
updated the PR